### PR TITLE
docs: Added Pull request template

### LINKED ISSUE #5: Docs: Missing PR template
**Is your feature request related to a problem? Please describe.**
The project does not have a PR template 

**Describe the solution you'd like**
A good PR template needs to be added for making Pull requests easier for everyone. 

**Additional context**
I would love to add a PR template if the issue is assigned to me

### PR DESCRIPTION
**Pull request by inkollusireeshaadharani**
## Proposed Changes
 
 Added PULL_REQUEST_TEMPLATE.md under .github folder.

### CODE REVIEW

**Review by ManishaSwain8 — DISMISSED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+<!-- Example: Closes #31 -->
+
+## Proposed Changes
+
+<!-- List all the changes you are proposing in this PR -->
+
+<!-- Mark the boxes as [x] if the statement is true, or [ ] if it's false or not applicable -->
+
+## Checklist (Check all applicable boxes)
+
```
Comment (ManishaSwain8) [inline .github/PULL_REQUEST_TEMPLATE.md:16]: Add check box for, if person is contributing independently or for hactoberfest2023

Comment (inkollusireeshaadharani) [reply .github/PULL_REQUEST_TEMPLATE.md:16]: sure!

**Comment by ManishaSwain8** [comment]
@inkollusireeshaadharani I have added a comment please check it out and update !

**Review by ManishaSwain8 — APPROVED** [review]

**Comment by inkollusireeshaadharani** [comment]
I have made the necessary changes @ManishaSwain8

**Comment by ManishaSwain8** [comment]
Thankyou for your contribution @inkollusireeshaadharani  !

Looking forward to more issues and collaborations.
If you learnt something from this repo make sure to ⭐ this repo.

Thankyou ,Happy coding !🚀

**Pull request merged**